packages/amplify-category-api/src/commands/api/override.ts

```diff
@@ -71,12 +70,19 @@ export const run = async (context: $TSContext) => {
     // Migration logic goes in here
     const apigwInputState = new ApigwInputState(context, selectedResourceName);
     if (!apigwInputState.cliInputsFileExists()) {
-      if (await prompter.yesOrNo(getMigrateResourceMessageForOverride(AmplifyCategories.API, selectedResourceName, false), true)) {
+      if (selectedResourceName === 'AdminQueries') {
+        const { dependsOn } = amplifyMeta[AmplifyCategories.API][selectedResourceName];
+        const props: AdminQueriesProps = {
+          apiName: selectedResourceName,
+          authResourceName: dependsOn.filter(entry => entry.category === AmplifyCategories.AUTH)[0].resourceName,
```

Comment:
nit: can we move this to a function , also handle the case where dependsOn is null/undefined